README.md

```diff
@@ -187,6 +187,8 @@ Other potentially useful environment variables may be found in `setup.py`.
 conda install cmake ninja
 # Run this command from the PyTorch directory after cloning the source code using the “Get the PyTorch Source“ section below
 pip install -r requirements.txt
+# (optional) If using torch.compile with inductor/triton, install the matching version of triton
+make triton
```

Comment:
Is this platform independent?

Reply:
it'll only work on linux, willl break on windows and mac

Reply:
Then should we move this to the Linux section below?

Reply:
oh i didn't realize triton is linux only.